charts/qdrant/templates/statefulset.yaml

```
@@ -113,7 +113,7 @@ spec:
             {{- end }}
             {{- if eq .name "http"}}
             httpGet:
-              path: /
+              path: /healthz
```

Shouldn't `readinessProbe` use `/readyz`? 

```suggestion
              path: /readyz
```

---

We currently have a startupProbe, readinessProbe and livenessProbe.

At the moment alle the endpoints are configured the same, but I guess at some point you want to have the possibility to have different implementations there(?).

What is the semantical difference between the endpoints?

---

Correct. In the current implementation they all function the same.

We'll eventually update this to work as described in <https://kubernetes.io/docs/reference/using-api/health-checks/>. The documentation is a bit vague however.

---

Ah,
> The healthz endpoint is deprecated (since Kubernetes v1.16)
I'll not use this than either here. It's probably still good to have a healthz endpoint that does the same as readyz.